Enable a list view for resources

glitchedmob opened the pull request:
Fixes #242 

Co-authored-by: Fred Lawler <fred@fredlawl.com>
Co-authored-by: Levi Zitting <me@levizitting.com>

readingdancer commented on covid-19-support/src/components/Sidebar.vue:
```diff
@@ -0,0 +1,221 @@
+
+#sidebar-wrapper {
+  transition: transform 0.25s ease-out;
+  transform: translateX(-100%);
```
I suggest changing this to:
`transform: translateY(100%);`

Then seem my next comment on line 134

readingdancer replied:
I suggest changing this to:
transform: translateY(100%);

This should allow the list and business details to slide up from the bottom, instead of in from the side.

Then seem my next comment on line 134

readingdancer commented on covid-19-support/src/components/Sidebar.vue:
```diff
@@ -0,0 +1,221 @@
+  }
+
+  @include media-breakpoint-up(md) {
+    height: auto;
```
If you add the `transform: translateX(-100%);` here, then everything should still work correctly on tablet & desktop. 

I.e. it will continue to slide in and out from the right hand side.

glitchedmob commented:
@readingdancer 

1. Fixed. The listing/detail should come up from the bottom now.
2. Fixed this as well. The whole map was being pushed way down. That doesn't happen anymore.
3. I've updated that button to say Show Resource Details if the detail view is open. I agree with your idea of showing a mini business profile there instead but I think that's best done in another PR. This one is already getting pretty big and it's somewhat holding up other progress

readingdancer commented:
Thank you @glitchedmob and @fredlawl for your work on the PR, now to create some more issue ticket to solve the rest of the little issues :)
